### DISCUSSION TITLE
0.1.0

### DISCUSSION THREAD

**Discussion by Kyjor** [Kyjor/JulGame.jl#85] [Announcements]
## What's Changed
* Fixed issue with physics timing by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/23
* Animation/Animator docs and function updates by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/24
* [131] Fixing editor bugs by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/25
* [135] Fix textbox in editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/26
* [39] Adding sprite layering by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/28
* [136] Platformer Collider by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/29
* Fixing colliders and rigidbodies not being deleted when destroyed by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/30
* Update script update and rendering order by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/31
* Add SDL function wrapper and update font loading by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/32
* [142] Sprite UI element by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/33
* [29] Scene switching in game mode by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/34
* Vector structures refactorization by @achengli in https://github.com/Kyjor/JulGame.jl/pull/36
* Fixed a few issues and added a test project to be used for profiling by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/37
* refactoring Math/Lerp.jl switching Number type to Real by @achengli in https://github.com/Kyjor/JulGame.jl/pull/38
* Feature/158 refactor component creation to use structs as dtos by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/40
* Optimize collision checking and debug drawing by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/42
* Added text back to screen button, fixed pathing, and added test by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/43
* Added pixel per unit support. By default, sprites will not use it by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/44
* Scene switching in editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/50
* Add SDL backend for editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/51
* add concrete types to main struct by @mrufsvold in https://github.com/Kyjor/JulGame.jl/pull/52
* Clean Up Imports by @mrufsvold in https://github.com/Kyjor/JulGame.jl/pull/55
* Using platformer to increase test coverage by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/56
* Creating game "window" for SDL editor renderer by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/53
* Refactor/deanonymize-get-prop-funcs by @mrufsvold in https://github.com/Kyjor/JulGame.jl/pull/57
* Refactor/dispatch-component-functions by @mrufsvold in https://github.com/Kyjor/JulGame.jl/pull/59
* Refactor/move call sites to functional by @mrufsvold in https://github.com/Kyjor/JulGame.jl/pull/60
* Feature/sdl editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/58
* Feature/using sdl gfx by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/62
* Continued updates to new SDL Renderer-Based Editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/61
* Feature/sdl editor continued updates by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/64
* Update editor to allow textbox editing by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/65
* We can load fonts in the editor now by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/66
* Updates to textbox in editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/67
* Create screen buttons in the editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/68
* Added ability to change font size by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/69
* Feature/main refactor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/70
* Display errors again in editor. Added save scene backup when crashing by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/71
* Remove get props, add more tests by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/72
* Feature/sprite crop by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/73
* Adding parenting in the editor with multi-select by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/74
* Improved scene view for the editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/75
* Reverted collision code change and updated function names by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/76
* Improved usability of editor by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/77
* Feature/duplication brush by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/78
* 0.1.0 Features and bug fixes by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/79
* 0.1.0 Bugfixes by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/80
* Improve collisions and rigidbody by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/81
* Add editor tooltips by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/83
* Bugfix/inactive handling by @Kyjor in https://github.com/Kyjor/JulGame.jl/pull/84

## New Contributors
* @achengli made their first contribution in https://github.com/Kyjor/JulGame.jl/pull/36
* @mrufsvold made their first contribution in https://github.com/Kyjor/JulGame.jl/pull/52

**Full Changelog**: https://github.com/Kyjor/JulGame.jl/compare/v0.1.0-alpha...0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Kyjor/JulGame.jl/releases/tag/0.1.0'>0.1.0 </a>.</em>

**Comment by mrufsvold** [comment]
Wow! Big day!

**Reply by Kyjor** [reply]
Yep! Thanks for the help getting us here! I plan on adding a write up to these notes as I'd like to call out some big wins that we can't see from the pr titles
